packages/theme/src/theme/_guidelines.scss

```diff
@@ -53,7 +53,7 @@ $navbar-brand-logo-width: 75px !default;
 /// Drawer z-index
 /// should always stay lower than dialog z-index which is set to 1040
 /// @type Number (Unitless)
-$drawer-z-index: 100 !default;
+$drawer-z-index: 15 !default;
```

Comment:
Not possible to use tokens ? 
Looks like tokens.$coral-elevation-layer-standard-front would fit 
<img width="275" alt="image" src="https://github.com/Talend/ui/assets/16574861/13535a59-120a-44dd-9fbd-04de5dc534e1">

Reply:
@Gbacc thank you, I updated code.